lib.js

```diff
@@ -0,0 +1,11 @@
+function getSubFromJWT(jwt) {
```

Comment:
Why wouldn't you commit this inside src, and let it be bablified like the rest of the code ?

Reply:
Good point, it should be there.